frida/src/device_manager.rs

```diff
@@ -15,29 +14,24 @@ use crate::Frida;
 use crate::Result;
 
 /// Platform-independent device manager abstraction access.
-pub struct DeviceManager<'a> {
+#[derive(Clone)]
+pub struct DeviceManager {
+    frida: Frida,
```

Comment:
we call this `frida` here, but `runtime()` above... Let's choose one and stick with it.